src/main/java/org/myrobotlab/arduino/virtual/MrlServo.java

```diff
@@ -110,7 +110,7 @@ public void update() {
         // There was a change in the currentPosition
         if (previousCurrentPosUs != (int) currentPosUs) {
           writeMicroseconds((int) currentPosUs);
-          publishServoEvent(SERVO_EVENT_POSITION_UPDATE);
+          publishServoEvent(SERVO_EVENT_STARTED);

```

Comment:
Kill this event.. it's not useful.  this will be written on every pass through the update loop!